src/Tools.cpp

```diff
@@ -0,0 +1,40 @@
+// C++ includes
+#include "Tools.hpp"
+#include <cmath>
+
+namespace normal_depth_map {
+
+double underwaterSignalAttenuation( const double frequency,
+                                    const double temperature,
+                                    const double depth,
+                                    const double salinity,
+                                    const double acidity) {
+
+    // borid acid and magnesium sulphate relaxation frequencies (in kHz)
+    double f1 = 0.78 * pow(salinity / 35, 0.5) * exp(temperature / 26);
+    double f2 = 42 * exp(temperature / 17);
+
+    // borid acid contribution
+    double borid = 0.106 * ((f1 * frequency * frequency) / (frequency * frequency + f1 * f1)) * exp((acidity - 8) / 0.56);
+
+    // magnesium sulphate contribuion
+    double magnesium = 0.52 * (1 + temperature / 43) * (salinity / 35)
+                        * ((f2 * frequency * frequency) / (frequency * frequency + f2 * f2)) * exp(-depth / 6000);
```

Comment:
In your code, you use many ` frequency * frequency `. I could compute at the begin of the method and keep  the result in other variable like : 
`double frequency2 = frequency * frequency;`

Reply:
Good practice!